permissions/src/commonMain/kotlin/dev/icerock/moko/permissions/DeniedExceptions.kt

```diff
@@ -0,0 +1,9 @@
+package dev.icerock.moko.permissions
+
+open class DeniedException(val permission: Permission, message: String?) : Exception(message) {
+    constructor(permission: Permission) : this(permission, null)
```

Comment:
you can just use default value at `message` arg:
```
open class DeniedException(val permission: Permission, message: String? = null) : Exception(message)
```

Reply:
Fixed.